src/uct/api/uct.h

```diff
@@ -1311,11 +1329,28 @@ UCT_INLINE_API void uct_ep_pending_purge(uct_ep_h ep, uct_pending_callback_t cb)
 
 /**
  * @ingroup UCT_RESOURCE
- * @brief
- */
-UCT_INLINE_API ucs_status_t uct_ep_flush(uct_ep_h ep)
+ * @brief Flush outstanding communication operations on an endpoint.
+ *
```

Comment:
I don't think we can distinguish between the two, unless you define separate communication channels. I think on UCT level it is ok to state the we flush everything. UCP level implements the semantics defined by MPI, as a result the flush behavior is a bit different. @yosefe FYI